data/coalition/coalition outfits.txt

```diff
@@ -171,6 +171,18 @@ outfit "Cooling Module"
 	"cooling energy" 1.4
 	description "This cooling module consumes energy. To conserve power, it does not ramp up to full strength unless a ship is very close to overheating."
 
+outfit "Fuel Module"
+	category "Systems"
+	licenses
+		Coalition
+	cost 250000
+	thumbnail "outfit/fuel module"
+	"mass" 19
+	"outfit space" -19
+	"fuel capacity" 400
+	description "In their early days of space exploration, the Saryds developed these to travel far away without worrying too much about their fuel."
```

Comment:
How about a nod to some of the lore?  Plus a minor stylistic suggestion...
```suggestion
	description "In their early days of space exploration, when Coalition space was neither well settled nor well connected, the Saryds developed the fuel module to travel far from Saros without worrying about becoming stranded without fuel."
```

Reply:
What lore is this a nod to?

Reply:
I have no idea what you plan on taking or leaving from the Document, but I recall it saying that before the war with the Quarg, Coalition space had fewer hyperlinks.  That's what I'm referencing with the first edit.  The remaining three just make it sound better without actually adding anything to the message.

Reply:
It does nothing but harm (or at least be an annoyance) to the Saryds if their space was initially cut to be a miniature maze, with fewer links than there are currently, and there would be no point in the Archons keeping it that way, or keeping the "clusters" of each species isolated form one another, after the Quarg gave the Saryds Jump Drives, so I'm probably just gonna ignore that.
I'll apply the other three changes in a bit.

Reply:
Right, that's your prerogative :-)

Personally I wasn't imagining an isolated maze though, just fewer hyperlinks, more like Human space is - Coalition space is extremely well-connected by comparison.  I imagine that some time in the future, once humans figure out the tech behind hyperlink creation, Human space will be just as well connected.